src/cosmosdb-preview/azext_cosmosdb_preview/_params.py

```diff
@@ -331,3 +331,37 @@ def load_arguments(self, _):
         c.argument('account_name', account_name_type, id_part=None, required=True, help='Name of the CosmosDB database account')
         c.argument('database_name', database_name_type, required=True, help='Name of the mongoDB database')
         c.argument('container_name', options_list=['--name', '-n'], required=True, help='Name of the mongoDB collection')
+
+    # Sql container partition retrieve throughput
+    with self.argument_context('cosmosdb sql container retrieve-partition-throughput') as c:
+        c.argument('account_name', account_name_type, id_part=None, required=True, help='Name of the CosmosDB database account')
+        c.argument('database_name', database_name_type, required=True, help='Name of the CosmosDB database name')
+        c.argument('container_name', options_list=['--name', '-n'], required=True, help='Name of the CosmosDB container')
+        c.argument('physical_partition_ids', options_list=['--physical-partition-ids', '-p'], required=False, help='comma separated list of physical partition ids')
```

Comment:
You can use `nargs=*` to allow customers provide a list of physical_partition_id